prime-router/docs/api/README.md

```diff
@@ -1,16 +1,79 @@
-This folder holds open api docs resources:
+# Swagger UI
+
+Swagger UI is a tool that allows you to visually interact with ReportStream APIs that have been defined using the OpenAPI
+Specification (OAS). It provides a user-friendly interface that makes it easier to understand and explore capabilities of
+the API without requiring external tools such as curl or Postman.
+
+This directory contains all open api spec resources:
 
 1. ./: specs in yaml format derived from other process (vs auto generated specs under ./generated)
 2. ./swagger-ui: swagger ui artifacts (with customization - see custom and maintain notes below)
 3. ./generated: auto generated api spec(s) from annotated kotlin source files
 
-swagger ui customization and maintainance:
+## Purpose
 
-swagger ui dist update history:
+The main purpose of Swagger UI is to:
 
-|    date    |  dist |              comments                |
-|------------|-------|--------------------------------------------------------------|
-| 07/24/2023 | 5.1.0 | only change made in swagger-initializer.js to prevent js injection |
+1. **Documentation:** generates documentation in a terse yet approachable format that serves as great reference
+   material.
+
+2. **API Visualization:** provides a structured interface that displays API endpoints, request parameters, response 
+   structures, and example requests and responses in a structured, readable, and navigable format.
+
+3. **API Testing:** facilitates direct interaction with the API. You can input parameters, make requests, and
+   see the responses right in your browser with no external tools.
+
+## Usage
+With an API selected the UI will display a list of endpoints. Each endpoint can be expanded to display the
+usage, parameters, and expected output; as well as an interface to make an API call.
+
+### Accessing the Swagger UI Interface
+
+The interface consists of static web assets that are compiled and served similarly to our front-end web application.
+The UI is served and accessible on the same server as the reportstream web ui / front-end using a particular URL / path.
+To access it, navigate to TLD for your chosen environment and append `/swagger-ui` to the end of the root url.
+
+**Staging** https://staging.prime.cdc.gov/swagger-ui
+**Local** https://localhost:8080/swagger-ui
```

Comment:
Does this link actually work?

Reply:
> Does this link actually work?

the URL on staging is to be decided, since just like front end - report stream page webUI - is serve through Akamai, and there is DNS masquerade to be done, e.g. front end on staging is on URL: https://staging.reportstream.cdc.gov/

but FE site is served from Azure Blob container: xxxxxxxxxxxx (obfuscated)

and it is accessible from URL: https://xxxxxxxxxxx.z13.web.core.windows.net/ obfustcated

Akamai thing will be delegated to DevOp since that's not in our scope